test/test_decomp.py

```diff
@@ -265,18 +267,20 @@ def normalize_op_input_output(f, sample, requires_grad=True):
         torch.bfloat16,
         "nn.functional.layer_norm",
     ),  # "batch_norm" not implemented for 'BFloat16'
-    ("cpu", torch.bfloat16, "addmm"),  # decomposition loses precision
-    ("cpu", torch.bfloat16, "softmax"),  # needs relaxed prec
-    ("cpu", torch.bfloat16, "log_softmax"),  # needs relaxed prec
     # CUBLAS_STATUS_NOT_SUPPORTED when calling
     # `cublasGemmStridedBatchedExFix(handle, opa, opb, (int)m, (int)n, (int)k,
     # (void*)&falpha, a, CUDA_R_16BF, (int)lda, stridea, b, CUDA_R_16BF,
     # (int)ldb, strideb, (void*)&fbeta, c, CUDA_R_16BF, (int)ldc, stridec,
     # (int)num_batches, CUDA_R_32F, CUBLAS_GEMM_DEFAULT_TENSOR_OP)`
     ("cuda", torch.bfloat16, "nn.functional.bilinear"),
+    # randomness
```

Comment:
oh, if the rng is the same in the decomp and the kernel we can fix this by just snapshotting rng state, maybe? IDK how practical it is to actually manage this. Feel free to delete the comment below this